### ISSUE TITLE
M1 bots red the tree with `bot died`

### ISSUE THREAD

**Issue by LongCatIsLooong** [flutter/flutter#103910]
Tree is closed due to purple builds:
https://ci.chromium.org/p/flutter/builders/prod/Mac_arm64_ios%20build_ios_framework_module_test

https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_arm64_ios%20build_ios_framework_module_test/382/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_arm64_ios%20build_ios_framework_module_test/381/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_arm64_ios%20build_ios_framework_module_test/380/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_arm64_ios%20build_ios_framework_module_test/379/overview

**Comment by keyonghan** [comment]
Based on https://github.com/flutter/flutter/issues/103522#issuecomment-1127934346, this is a different issue, and it is still happening.

Another recent failure: https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_arm64_ios%20macos_chrome_dev_mode/385/overview

**Comment by LongCatIsLooong** [comment]
Observing similar failures this morning https://ci.chromium.org/p/flutter/builders/prod/Mac_arm64_ios%20build_ios_framework_module_test

**Comment by yusuf-goog** [comment]
Following up with the Luci team to try and get logs. This is definitely a new issue and not what we were seeing before.

**Comment by keyonghan** [comment]
> Following up with the Luci team to try and get logs. This is definitely a new issue and not what we were seeing before.

Taking a look at the mac42 events, seems it still keeps rebooting around every 5 mins
<img width="1113" alt="Screen Shot 2022-05-17 at 10 02 41 AM" src="https://user-images.githubusercontent.com/54558023/168870404-269659f0-64b6-4f82-97d1-ca5afa08d769.png">

**Comment by keyonghan** [comment]
3 of 4 M1 bots are hitting this issue, and manually quarantined them to skip running builds.
Re-triggered the failed test, and it should be green soon by running on mac44.

**Comment by yusuf-goog** [comment]
There were 2 swarming_bot processes starting up each time. One of the tried to get the swarming lock file, failed and set the host to reboot.

Looks like it was some launchctl related issue. We tried a number of things to stop the launchctl from starting up any swarming_bot process. Moving the plist file out, unloading etc. only caused one of the processes to stop.

Then we disabled the task:

```sudo launchctl disable user/502/org.swarm.bot.plist```

and that seemed to stop the single process from running as well. After disabling it, and reloading the plist file, it seems to now be working, with only 1 swarming_bot process showing up.
